Description: Backwards-compat for cargo 0.47

  The flag being removed here was added in
  https://github.com/rust-lang/cargo/pull/9404 released in cargo 0.54
  
  This works around a feature introduced in this PR
  https://github.com/rust-lang/cargo/pull/8640 released in cargo 0.53
  
  Therefore it is not needed for Debian's current cargo 0.47.
  
  We can drop this patch when updating to cargo 0.54 and later.

--- a/src/bootstrap/doc.rs
+++ b/src/bootstrap/doc.rs
@@ -443,7 +443,6 @@ impl Step for Std {
             cargo
                 .arg("-p")
                 .arg(package)
-                .arg("-Zskip-rustdoc-fingerprint")
                 .arg("--")
                 .arg("--markdown-css")
                 .arg("rust.css")
@@ -589,7 +588,6 @@ impl Step for Rustc {
         cargo.rustdocflag("--generate-link-to-definition");
         compile::rustc_cargo(builder, &mut cargo, target);
         cargo.arg("-Zunstable-options");
-        cargo.arg("-Zskip-rustdoc-fingerprint");
 
         // Only include compiler crates, no dependencies of those, such as `libc`.
         // Do link to dependencies on `docs.rs` however using `rustdoc-map`.
@@ -722,7 +720,6 @@ macro_rules! tool_doc {
                     &[],
                 );
 
-                cargo.arg("-Zskip-rustdoc-fingerprint");
                 // Only include compiler crates, no dependencies of those, such as `libc`.
                 cargo.arg("--no-deps");
                 $(
